pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -429,18 +431,28 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 	var istioGenerated bool
 	var err error
 
+	// For mounted certificates (if any) determine the set of files.
+	// If not found - we'll load from K8S Secret.
 	fileBundle, err := detectSigningCABundle()
 	if err != nil {
 		return nil, fmt.Errorf("unable to determine signing file format %v", err)
 	}
+	// the fileBundle will have file paths - but they may not exist.
+	// TODO: if no file is mounted we may fallback to loading from secrets.
+
 	if _, err := os.Stat(fileBundle.SigningKeyFile); err == nil {
 		detectedSigningCABundle = true
 		if _, err := os.Stat(path.Join(LocalCertDir.Get(), ca.IstioGenerated)); err == nil {
 			istioGenerated = true
 		}
 	}
+	// TODO: if the cert chain length > 1 - we can't refresh so we can use the other
```

Comment:
seems we donot refresh here, why docment it

Reply:
I meant: the code here doesn't seem to do any refresh, the TODO is to remove this code ( assuming reviewers don't find a reason to keep it ), and use the other code that does refresh.

Since I don't fully understand the use case behind this portion or if there is some code that actually does refresh - left it as a TODO.